backend/ohq/management/commands/membership_stat.py

```diff
@@ -0,0 +1,31 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+
+from ohq.models import Membership, Question
+from ohq.statistics import calculate_instructor_avg_time_stats, calculate_student_avg_time_stats
+
+
+class Command(BaseCommand):
+    def calculate_statistics(self, users):
+
+        # Iterate through all the users, building membership stats for each user
+        for user in users:
+            memberships = Membership.objects.filter(user=user, course__archived=False)
+            for membership in memberships:
+                questions_asked = Question.objects.filter(
```

Comment:
Checked with Ying on this and we'll just stick to questions that were asked, especially since rejected questions make up a small fraction of questions anyways